x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/metric/SecurityMetricType.java

```diff
@@ -19,6 +19,25 @@ public enum SecurityMetricType {
         new SecurityMetricInfo("es.security.authc.api_key.time", "Time it took (in nanoseconds) to execute API key authentication.", "ns")
     ),
 
+    AUTHC_SERVICE_ACCOUNT(
```

Comment:
nit: (likely a different PR) do you think it is worth the effort to enforce some of these naming conventions via a unit test ?

Reply:
It makes total sense. I'll add it in a followup PR.